fiatdao/fiat-ui
# Redeem matured assets

## Issue ticket number & link
no ticket but report here: https://discord.com/channels/931941263371042866/941434334886375436/1000047906155401276

## Solution
* Fix runtime error for User who had LUSD locked up. should be able to withdraw collateral
* Added support for redeem but not confident it will work so hid it from ui behind show_underlying flag

## QA Instructions
* Withdraw from a matured position
* Open expired LUSD position on forked mainnet and ensure user sees withdraw tab
